cloud/service-operations/service-management.md

```diff
@@ -40,18 +40,24 @@ data discrepancy between services.
 
 ### Forking a service
 
-1.  In the Timescale Cloud console, from the `Services` list, ensure the service you want to form has a status of `Running` or `Paused`, then click the name of
-    the service you want to fork.
+1.  In the Timescale Cloud console, from the `Services` list, ensure the service
+    you want to form has a status of `Running` or `Paused`, then click the name
+    of the service you want to fork.
 1.  Navigate to the `Operations` tab.
-1.  In the `General` section, click `Fork service`. In the dialog, confirm by
-    clicking `Fork service`. The forked service takes a few minutes to start.
-1.  To change the configuration of your fork, click
-    `Advanced options`. You can set different compute and
-    storage options, separate from your original database.
+1.  In the `Service management` section, click `Fork service`. In the dialog,
+    confirm by clicking `Fork service`. The forked service takes a few minutes
+    to start.
+1.  **(optional)** To change the configuration of your fork, click `Advanced
```

Comment:
```suggestion
1.  **OPTIONAL** To change the configuration of your fork, click `Advanced
```

We might need to add this to the style guide. I think we do it differently every time.